### ISSUE TITLE
Particle indicator for furnitures larger than one block

### ISSUE THREAD

**Issue by ItzASecret** [PluginBugs/Issues-ItemsAdder#2911]
### Terms

- [X] I'm using the very latest version of ItemsAdder and its dependencies.
- [X] I already searched on this [Github page](https://github.com/PluginBugs/Issues-ItemsAdder/issues) to check if the same suggestion was already reported.
- [X] I already searched on the [plugin wiki](https://itemsadder.devs.beer/) to know if the feature is already implemented.
- [X] I already searched on the **Discord** server to know if anyone already has a solution for this or if the feature was already suggested.

### Discord tag (optional)

itzasecretboi

### Describe the solution you'd like

We can make furnitures larger than one block with IA!

The only issue is, we're never sure quite how it's going to be aligned, leading to us usually breaking and replacing the item a few times.

So I was thinking, what if we detect if we have a furniture that is larger than one block, meaning it's width is larger than 1, and if it is, we outline block shaped particles where the player's furniture would normally be placed.

This would help the player see how it would be placed and how many blocks wide it is before placing it.

This is just a QOL feature, and a random idea I had, i'm not sure if you can actually outline particles on blocks based on where the player is looking, but seemed like a good and simple concept to me.

### Is your feature request related to a problem?

We're never sure quite how large furnitures are going to be aligned, leading to us usually breaking and replacing the item a few times.

### Describe alternatives you've considered

None

### Additional context

![image](https://github.com/PluginBugs/Issues-ItemsAdder/assets/82130468/65dff744-a54a-4c7a-afe1-b19396bd604c)
 
 I'm imagining something like this, except around where the block would be placed.

**Comment by LoneDev6** [comment]
Basically previewing the furniture while holding it in hand?
Maybe while pressing SHIFT in order to trigger the functionality only on demand.

**Comment by ItzASecret** [comment]
> Basically previewing the furniture while holding it in hand? Maybe while pressing SHIFT in order to trigger the functionality only on demand.

Yep, exactly that. Pressing shift to preview it would pretty much make it perfect.
